pkg/state/impl/etcd/etcd.go

```diff
@@ -587,12 +573,16 @@ func (st *State) watchKind(ctx context.Context, resourceKind resource.Kind, sing
 	watchCh := st.cli.Watch(ctx, etcdKey, clientv3.WithPrefix(), clientv3.WithPrevKV(), clientv3.WithRev(revision))
 
 	go func() {
+		clearbootstrapList := func() { bootstrapList = nil }
+
 		defer func() {
+			clearbootstrapList()
```

Comment:
I don't think I understand the goal of this change ?

Reply:
It's a leftover from my experimentations on removing the memory leak. Removed.